File: convert_charcode/convert_charcode.pl
use Encode::EUCJPMS;
use URI::Escape;
my $utf8str = "車";
my $eucstr = $utf8str;
Encode::from_to($eucstr, "utf8", "eucjp-ms");
print uri_escape($eucstr)."\n";
$utf8str = $eucstr;
Encode::from_to($utf8str, "eucjp-ms", "utf8");
print($utf8str."\n");

File: convert_charcode/convert_charcode.php
<?php
$utf8str = "車";
$eucstr = mb_convert_encoding( $utf8str, 'euc-jp', 'utf8');
print rawurlencode($eucstr)."\n";
$utf8str = $eucstr;
$utf8str = mb_convert_encoding( $utf8str, 'utf8', 'euc-jp');
print $utf8str."\n";
